glances/plugins/containers/engines/docker.py

```diff
@@ -215,25 +215,26 @@ class DockerContainersExtension:
 
     CONTAINER_ACTIVE_STATUS = ['running', 'paused']
 
-    def __init__(self):
+    def __init__(self, server_urls:str|list = 'unix:///var/run/docker.sock'):
         if import_docker_error_tag:
             raise Exception("Missing libs required to run Docker Extension (Containers) ")
 
-        self.client = None
+        self.clients = None
         self.ext_name = "containers (Docker)"
         self.stats_fetchers = {}
 
-        self.connect()
+        self.connect(server_urls)
 
-    def connect(self):
+    def connect(self, base_urls:str|list = 'unix:///var/run/docker.sock'):
         """Connect to the Docker server."""
         # Init the Docker API Client
+        base_urls = [base_urls] if isinstance(base_urls, str) else base_urls
         try:
             # Do not use the timeout option (see issue #1878)
-            self.client = docker.from_env()
+            self.clients = [docker.DockerClient(url) for url in base_urls]
```

Comment:
This will end up failing/dropping all socket connections if any one of them fail.

I think a more tolerant approach where we just ignore the connections that fail would be better as they user will still get to see some of the containers if they are active.

A log message indicating the failed socket would be helpful for debugging